packages/storybook-builder-vite/codegen-iframe-script.ts

```diff
@@ -24,6 +25,11 @@ export async function generateIframeScriptCode(options: ExtendedOptions) {
   const configEntries = [...presetEntries, previewOrConfigFile].filter(Boolean);
 
   const storyEntries = await listStories(options);
+  const resolveMap = storyEntries.reduce(
```

Comment:
Without specifying a generic for `reduce`, typescript will infer the type of `resolveMap` based on the second argument, which is an empty object.  Since we know the result of the reduce is an object with string keys and string values, we can use this:

```suggestion
  const resolveMap = storyEntries.reduce<Record<string, string>>(
```